pyvista/plotting/plotting.py

```diff
@@ -1136,38 +1154,55 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
                 * ``decimate``: ``float`` between 0 and 1, level of decimation
 
         use_transparency : bool, optional
-            Invert the opacity mappings and make the values correspond to
-            transparency.
+            Invert the opacity mappings and make the values correspond
+            to transparency.
 
         below_color : string or 3 item list, optional
-            Solid color for values below the scalars range (``clim``). This
-            will automatically set the scalar bar ``below_label`` to
-            ``'Below'``
+            Solid color for values below the scalars range
+            (``clim``). This will automatically set the scalar bar
+            ``below_label`` to ``'Below'``
 
         above_color : string or 3 item list, optional
-            Solid color for values below the scalars range (``clim``). This
-            will automatically set the scalar bar ``above_label`` to
-            ``'Above'``
+            Solid color for values below the scalars range
+            (``clim``). This will automatically set the scalar bar
+            ``above_label`` to ``'Above'``
 
         annotations : dict, optional
-            Pass a dictionary of annotations. Keys are the float values in the
-            scalars range to annotate on the scalar bar and the values are the
-            the string annotations.
+            Pass a dictionary of annotations. Keys are the float
+            values in the scalars range to annotate on the scalar bar
+            and the values are the the string annotations.
 
-        pickable : bool
+        pickable : bool, optional
             Set whether this mesh is pickable
 
         log_scale : bool, optional
-            Use log scale when mapping data to colors. Scalars less than zero
-            are mapped to the smallest representable positive float. Default:
-            ``True``.
+            Use log scale when mapping data to colors. Scalars less
+            than zero are mapped to the smallest representable
+            positive float. Default: ``True``.
+
+        pbr : bool, optional
+            Enable physics based rendering (PBR) if the mesh is
+            ``PolyData``.  Use the ``color`` argument to set the base
+            color. This is only available in VTK>=9.
+
+        metallic : float
+            Usually this value is either 0 or 1 for a real material
+            but any value in between is valid. This parameter is only
+            used by PBR interpolation. Default value is 0.0
+
+        roughness : float
+            This value has to be between 0 (glossy) and 1 (rough). A
+            glossy material has reflections and a high specular
+            part. This parameter is only used by PBR
+            Interpolation. Default value is 0.5
```

Comment:
Interpolation can also be uppercased, but then we should change the description of `metallic` right above.
```suggestion
            interpolation. Default value is 0.5.
```

Reply:
Just make them both lowercase.